modules/kernel/src/main/scala/org/tessellation/kernel/package.scala

```diff
@@ -13,7 +13,6 @@ package object kernel {
   type KernelKryoRegistrationId = KryoRegistrationId[KernelKryoRegistrationIdRange]
 
   val kernelKryoRegistrar: Map[Class[_], KernelKryoRegistrationId] = Map(
-    classOf[ProcessSnapshot] -> 400,
-    classOf[StateChannelSnapshotGist] -> 401
```

Comment:
Shouldn't we somehow mark that the 401 was already used, to avoid the situation where we reuse that registration id for another class? Nothing probably will happen in this particular case, but I wonder if it shouldn't be the approach in general to register new classes under new (never used) ids only.

Reply:
I think we should distinguish `never used` from `never serialized`. If it hasn't been serialized with these ids before then we should be fine. If you want to mark anyway then we can do some refined rule like `Not[400]` or sth.

Reply:
Just pointing to a possible problem. It may also not be that easy at some point to determine if a class was ever serialized or not.